CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+<<<<<<< HEAD
+# v1.8.1 (2017-07-24)
```

Comment:
You've left conflict marker in ;)
Also — can you specify the version as `HEAD` or `Unreleased`? The actual version number will be set by the maintainer (me, probably) when reviewing what was merged and deciding whether this needs to be a major or a minor.